components/Footer.js

```diff
@@ -1,14 +1,46 @@
 import React from 'react';
+import FooterLink from './FooterLink';
 
-const Footer = () => {
+const FooterWrapper = (props) => {
+  const style = {
+    backgroundColor: '#D2D3DC',
+    border: 'solid 1px #D2D3DC',
+    boxShadow: '0 0 5px #777',
+    padding: 15,
```

Comment:
For this padding - do you mean 15em, or 15px? Without the 'px' specification I think it will assume you mean 'em', which will be a ton of padding, no?